src/backend/blockchain/tests/unit/test_functions.py

```diff
@@ -0,0 +1,112 @@
+from src.transaction import Transaction
```

Comment:
This is a duplicate, something went weird in the merge. It will be deleted on the next commit